meta-lmp-bsp/recipes-bsp/u-boot/u-boot-fio/imx8mn-lpddr4-evk-sec/lmp.cfg

```diff
@@ -0,0 +1,120 @@
+CONFIG_SPL_DM=y
+CONFIG_SPL_FIT=y
+CONFIG_SPL_FIT_SIGNATURE=y
+CONFIG_SPL_FIT_SIGNATURE_STRICT=y
+CONFIG_SPL_LOAD_FIT=y
+# CONFIG_SPL_FIT_IMAGE_TINY is not set
+# CONFIG_SPL_BOOTROM_SUPPORT is not set
+CONFIG_SPL_ALLOC_BD=y
+CONFIG_SPL_OF_CONTROL=y
+CONFIG_SPL_CRYPTO=y
+CONFIG_SPL_HASH=y
+CONFIG_SPL_SHA256_SUPPORT=y
+CONFIG_SPL_WATCHDOG=y
+CONFIG_SPL_HW_WATCHDOG=y
+CONFIG_RSA=y
+CONFIG_HASH=y
+CONFIG_SPL_ATF=y
+CONFIG_SPL_MMC=y
+# CONFIG_SPL_LEGACY_IMAGE_SUPPORT is not set
+# CONFIG_SPL_DM_USB is not set
+# CONFIG_SPL_EFI_PARTITION is not set
+# CONFIG_SPL_PARTITION_UUIDS is not set
+# CONFIG_SPL_DOS_PARTITION is not set
+# CONFIG_SPL_MXC_OCOTP is not set
+# CONFIG_SPL_USB_HOST is not set
+# CONFIG_SPL_USB_GADGET is not set
+# CONFIG_SPL_USB_SDP_SUPPORT is not set
+# CONFIG_SPL_WDT is not set
+# CONFIG_SPL_SHA_HW_ACCEL is not set
+# CONFIG_CMD_DEKBLOB is not set
+CONFIG_OF_LIBFDT_OVERLAY=y
+CONFIG_FIT=y
+CONFIG_FIT_SIGNATURE=y
+CONFIG_FIT_SIGNATURE_STRICT=y
+CONFIG_FIT_VERBOSE=y
+CONFIG_SECONDARY_BOOT_RUNTIME_DETECTION=y
+# TODO: fix hard-coded secondary boot offset
+# defined in fuse 0x490 [22:19] IMG_CNTN_SET1_OFFSET
+# default value is 0x0 == 4MB offset
+CONFIG_SECONDARY_BOOT_SECTOR_OFFSET=0x1000
+# CONFIG_LEGACY_IMAGE_FORMAT is not set
+CONFIG_PARTITION_UUIDS=y
+CONFIG_TEE=y
+CONFIG_OPTEE=y
+CONFIG_OPTEE_LIB=y
+CONFIG_FIOVB=y
+CONFIG_CMD_FIOVB=y
+CONFIG_CMD_XIMG=y
+CONFIG_CMD_SECONDARY_BOOT=y
+CONFIG_SUPPORT_EMMC_RPMB=y
+CONFIG_SUPPORT_EMMC_BOOT=y
+# CONFIG_ENV_IS_IN_MMC is not set
+# CONFIG_ENV_IS_IN_FAT is not set
+CONFIG_ENV_IS_NOWHERE=y
+CONFIG_ENV_FAT_INTERFACE="mmc"
```

Comment:
I think these `CONFIG_ENV_FAT_*` settings can be removed from the `-sec`

Reply:
Sure, done.